ensime-startup.el

```diff
@@ -117,6 +117,25 @@ that you have read this message.")
   (let ((ensime-prefer-noninteractive t))
     (ensime--maybe-update-and-start orig-buffer-file-name)))
 
+(defun is-dev-version? (version)
+    "It check VERSION string for few patterns coresponded to dev server version string format."
+    (-contains?
+     (-map (lambda (s) (s-contains? s version))
+           '("-M" "-RC" "SNAPSHOT")
+           )
+     t)
+    )
+
+(defun is-server-jars-has-dev-version? (server-jars)
```

Comment:
we don't need this anymore. In fact, if there is no `:ensime-server-version` field in the `.ensime` we should warn that the user's build tool plugin is out of date.